cmd/neofs-cli/docs/storage-node-xheaders.md

```diff
@@ -0,0 +1,29 @@
+# Extended headers
+
+## Overview
+
+Extended headers are used for Request/Response. They may contain any user-defined headers
+to be interpreted on application level.
+Key name must be a unique valid UTF-8 string. Value can't be empty. Requests or
+Responses with duplicated header names or headers with empty values will be
+considered invalid.
+
+## Existing headers
+
+There are some "well-known" headers starting with `__NEOFS__` prefix that
+affect system behaviour:
+
+* `__NEOFS__NETMAP_EPOCH` - netmap epoch to use for object placement calculation. The `value` is string
+encoded `uint64` in decimal presentation. If set to '0' or not set, the
+current epoch only will be used.
+* `__NEOFS__NETMAP_LOOKUP_DEPTH` - if object can't be found using current epoch's netmap, this header limits
```

Comment:
I would add that this depth is applied to a current epoch _or the value of `__NEOFS__NETMAP_EPOCH` attribute_ .

Reply:
Updated.